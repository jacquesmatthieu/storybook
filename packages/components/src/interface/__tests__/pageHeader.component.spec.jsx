import React from 'react';
import { shallow } from 'enzyme';

import PageHeader from '../pageHeader.component';

describe('PageHeader', () => {
  let props;

  beforeEach(() => {
    props = {
      steps: ['foo', 'bar'],
      title: 'Foo',
    };
  });

  const getWrapper = () => shallow(<PageHeader {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
