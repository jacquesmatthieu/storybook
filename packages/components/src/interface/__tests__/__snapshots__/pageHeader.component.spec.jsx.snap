// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader should render properly 1`] = `
<pageHeadercomponent__PageHeaderWrapper
  centered={false}
  className=""
  vertical={false}
>
  <div>
    <WithCurrentApp(Breadcrumb)
      location={
        Object {
          "pathname": "foo",
        }
      }
      match={
        Object {
          "params": Object {},
        }
      }
      steps={
        Array [
          "foo",
          "bar",
        ]
      }
    />
    <typographycomponent__Typography
      variant="headline"
    >
      Foo
    </typographycomponent__Typography>
  </div>
</pageHeadercomponent__PageHeaderWrapper>
`;
